Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -19,6 +19,10 @@
 #include "DolphinQt2/Settings.h"
 #include "InputCommon/InputConfig.h"
 
+#ifdef USE_DISCORD_PRESENCE
```

Comment:
Why is this included here if it's not used for anything?

Reply:
I was removing all my code from the file but forgot to remove that while in the process.